### PR TITLE
[SVD] fix a bug when passing image as tensor

### PR DESCRIPTION
**Pull request by yiyixuxu**
fix https://github.com/huggingface/diffusers/issues/6574

when user passes `image` input as a tensor with range `[0,1]` (not normalized), we still have to pass it through the CLIP Image Processor before encode it with image encoder 

this script now works as expected
```python
import torch
from diffusers import StableVideoDiffusionPipeline
from diffusers.utils import load_image, export_to_video

pipe = StableVideoDiffusionPipeline.from_pretrained(
    "stabilityai/stable-video-diffusion-img2vid-xt", torch_dtype=torch.float16, variant="fp16"
)
pipe.enable_model_cpu_offload()

# Load the conditioning image
image = load_image("https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/svd/rocket.png")
image = image.resize((1024, 576))
image = pipe.image_processor.pil_to_numpy(image)  # to np
image = pipe.image_processor.numpy_to_pt(image)  # to pt


generator = torch.manual_seed(42)
frames = pipe(image, decode_chunk_size=8, generator=generator).frames[0]
export_to_video(frames, "out.mp4", fps=7)
```

https://github.com/huggingface/diffusers/assets/12631849/9e51c386-95d2-4973-b134-e7e24102524e

### CODE REVIEW

**Review by DN6 — APPROVED** [review]
Nice! 👍🏽
